frontend/src/pages/Register.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import Col from 'react-bootstrap/Col';
+
+
+export default function Register(){
```

Comment:
Should be like this:
```suggestion
export default function Register() {
```